### PR TITLE
New publish local script

### PR DESCRIPTION
**Pull request by acunniffe**

### CODE REVIEW

File: publish-locally.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  then
+    echo "No version provided"
+    exit 1
+fi
+
+
+yarn run bump "$1"
+
+yarn install
+source ./export_development_aliases.sh
```
Comment (devdoshi) [inline publish-locally.sh:13]: Does this line do anything?

Comment (devdoshi) [reply publish-locally.sh:13]: oh I guess it exports $OPTIC_SRC_DIR for below?

Comment (acunniffe) [reply publish-locally.sh:13]: Yes

**Comment by acunniffe** [comment]
Had some funky git history -- sorry! Doing a rare squash

**Pull request merged**